addons/website_blog/static/src/js/website_blog.editor.js

```diff
@@ -228,11 +236,26 @@ options.registry.blog_cover = options.Class.extend({
 
         _.each(this.$el, function (el) {
             var $el = $(el);
-            $el.toggleClass('d-none',
-                $el.is(':not([data-change])') && !self.$target.hasClass('cover')
-                || $el.is(':has([data-select-class])') && self.$target.hasClass('o_list_cover'));
+
+            if ($el.is(':not([data-change])')) {
```

Comment:
= `!$el.is('[data-change]')`
= `!$el.attr('data-change')`
= `!$el.data('change')` ? (maybe)